src/cript/api/api.py

```diff
@@ -258,6 +259,33 @@ def _get_vocab(self) -> dict:
 
         return self._vocabulary
 
+    # TODO needs to work with enums instead of strings
+    def get_vocab_by_category(self, category: str) -> List[dict]:
```

Comment:
I think it would be best to have this not contact the API all the time.
This is probably going to be called a lot if  we use if for material deserialization and we could like over load the back end.

We have a mechanism to cache the vocab already, it should use that instead.
(Even for a first version, let's not do the work multiple times.)

Reply:
You are 100% right, great catch, let me add that in

Reply:
done! ✅